anyway/infographics_utils.py

```diff
@@ -281,12 +283,10 @@ def filter_and_group_injured_count_per_age_group(data_of_ages):
 
 
 def get_most_severe_accidents_table_title(location_info):
-    return "תאונות בסדר חומרה יורד במקטע " + location_info["road_segment_name"]
-
+    return _('Accidents in descending order by severity for section {}').format(location_info['road_segment_name'])
```

Comment:
let's do f-string format:
`f"Accidents in descending order by severity for section { location_info['road_segment_name'] }"`,
I think it will make it more readable, what do you think?

Reply:
applies to all changes below.

Reply:
Cool, I'll adjust.